portfolio.html

```diff
@@ -8,6 +8,284 @@
+            <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-placeholder" alt="image placeholder">
+            <div class="desktop-languages">
+                <img src="./images/mobile_version/Card/Icon-languages.png" class="desktop-language-icon" alt="languages icon">
+            </div>
+            <div class="desktop-languages-list">
+            <h1 class="desktop-languages-header">Languages</h1>
+                <ul class="desktop-title-and-tags">
+                    <li class="lang-1">JavaScript</li>
+                    <li class="lang-2">Ruby</li>
+                    <li class="lang-3">Html</li>
+                    <li class="lang-4">Css</li>
+                </ul>
+            </div>
+        </div>    
+    
+        <div class="desktop-framework-cards">
+            <div class="desktop-frameworks">
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-placeholder2" alt="image placeholder">
+                <div class="desktop-framework-tags">
+                    <img src="./images/mobile_version/Card/Icon-frameworks.png" class="desktop-framework-icon" alt="frameworks icon">
+                </div>
+                <div class="desktop-frameworks-list">
+                    <h1 class="desktop-framework-header">Frameworks</h1>
+                <ul class="desktop-frame-tags">
+                    <li class="frame-19">Bootstrap</li>
+                    <li class="frame-20">Ruby on rails</li>
+                    <li class="frame-21">RSpec</li>
+                    <li class="frame-22">Capybara</li>
+                    <li class="frame-23">Selenium</li>
+                </ul>
+                </div>
+            </div>
+        </div>
+        
+        <div class="desktop-skills-cards">
+            <div class="desktop-skills">
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-placeholder3" alt="image placeholder">
+                <div class="desktop-skills-tags">
+                    <img src="./images/mobile_version/Card/Icon-skills.png" class="desktop-skills-icon" alt="skills icon">
+                </div>
+                <div class="desktop-skills-list">
+                    <h1 class="desktop-skills-header">Skills</h1>
+                <ul class="desktop-skills-frame">
+                    <li class="skills-24">Codekit</li>
+                    <li class="skills-25">GitHub</li>
+                    <li class="skills-26">Codepen</li>
+                    <li class="skills-27">Gitlab</li>
+                    <li class="skills-28">Terminal</li>
+                </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+    </section>
+
+    <section class="desktop-form">
+        <div class="desktop-form-primary-text">
+            <h2 class="desktop-form-text">I'm always interested in hearing about new projects, so if you'd <br> like to chat please get in touch.</h2>
```

Comment:
- [ ] Check the border colors of the form and adjust them according to the desgin
![image](https://user-images.githubusercontent.com/62705947/186937833-a490f44e-6cff-43ac-ab4a-33bd3e67241b.png)